packages/common/src/errors.ts

```diff
@@ -13,4 +13,6 @@
 export const enum RuntimeErrorCode {
   // NgSwitch errors
   PARENT_NG_SWITCH_NOT_FOUND = 2000,
+  // Pipe errors
+  INVALID_PIPE_ARGUMENT = 3000
```

Comment:
According to the message above, 2000-2999 range is reserved for the Common package, so we can use 2100-2199 range for pipe errors:
```suggestion
  INVALID_PIPE_ARGUMENT = 2100,
```

Reply:
Ops my bad, I missed checking the above comment. Will correct it. 👍